code/__DEFINES/dcs/signals.dm

```diff
@@ -861,3 +861,5 @@
 #define COMSIG_OBJECTIVE_CHECK_VALID_TARGET "objective_check_valid_target"
 	#define OBJECTIVE_VALID_TARGET		(1<<0)
 	#define OBJECTIVE_INVALID_TARGET	(1<<1)
+
+#define COMSIG_PREVENT_DELAYED_KNOCKDOWN "prevent_delayed_knockdown"
```

Comment:
I would call this `COMSIG_LIVING_CLEAR_STUNS`

Reply:
but those are not stuns, and we're not clearing em, we're clearing currently delayed knockdown here, are you **_sure_**?

Reply:
no, you are sending a signal that happens when you clear stuns. 

we are naming a signal not a proc call.